SimPEG/Survey.py

```diff
@@ -206,6 +206,54 @@ def fromvec(self, v):
                 self[src, rx] = v[indBot:indTop]
                 indBot += rx.nD
 
+class Data(BaseData):
+    """
+    Storage of data, standard_deviation and floor storage
+    with fancy [Src,Rx] indexing.
+
+
+
+    """
+
+    def __init__(self, survey, v=None, standard_deviation=None, floor=None):
+        # Initiate the base problem
+        BaseData.__init__(self, survey, v)
+
+        # Set the uncertainty parameters
+        # Note: Maybe set these
+        self.standard_deviation = StandardDeviation(self.survey, standard_deviation)
+        self.floor = Floor(self.survey, floor)
+
+
+    def eval(self):
```

Comment:
can we rename this?

Here we are not evaluating our data, we are making synthetic?